### DISCUSSION TITLE
A Camel named jeremy, I think

### DISCUSSION THREAD

**Discussion by Pengu-94** [gamesforklaus/brainfluff#2] [Creation]
![image](https://user-images.githubusercontent.com/77797271/230665310-e237128d-d41e-40a8-b5e4-2e29d85cf00e.png)

My creation.
His name is jeremy.
Please be nice to jeremy.

**Comment by gamesforklaus** [comment]
made me laugh... made me cry... my time with jeremy has been quite the roller-coaster ride

**Comment by Arcadius2006** [comment]
nice to meet you, Jerry
